node-operators/appchain-node/rpc-systemd.md

```diff
@@ -0,0 +1,265 @@
+mv ./westend-alphanet-raw-specs.json /var/lib/appchain-data
+```
+
+!!! note
+    To keep all the necessary files grouped in the same directory, it is recommended to also copy there your Appchain's specification file.
+
+### Create the Systemd Service configuration file {: #create-systemd-configuration }
+
+The next step is to create the Systemd configuration file. 
+
+You can create the file by running the following command:
+
+```bash
+sudo touch /etc/systemd/system/appchain.service
+```
+
+Now you can open the file using your favorite text editor (vim, emacs, nano, etc) and add the configuration for the service:
+
+```bash
+[Unit]
+Description="Appchain systemd service"
+After=network.target
+StartLimitIntervalSec=0
+
+[Service]
+Type=simple
+Restart=on-failure
+RestartSec=10
+User=appchain_node_service
+SyslogIdentifier=appchain
+SyslogFacility=local7
+KillSignal=SIGHUP
+ExecStart=/var/lib/appchain-data/container-chain-template-APPCHAIN_TYPE-node \
+--chain=YOUR_APPCHAIN_SPECS_FILE_LOCATION \
+--base-path=/var/lib/appchain-data \
+--rpc-port=9944 \
+--name=para \
+--bootnodes=INSERT_YOUR_APPCHAIN_BOOTNODE \
+-- \
+--chain=/var/lib/appchain-data/westend-alphanet-raw-specs.json \
+--rpc-port=9945 \
+--name=relay \
+--sync=fast \
+--bootnodes=/dns4/frag3-stagenet-relay-val-0.g.moondev.network/tcp/30334/p2p/12D3KooWKvtM52fPRSdAnKBsGmST7VHvpKYeoSYuaAv5JDuAvFCc \
+--bootnodes=/dns4/frag3-stagenet-relay-val-1.g.moondev.network/tcp/30334/p2p/12D3KooWQYLjopFtjojRBfTKkLFq2Untq9yG7gBjmAE8xcHFKbyq \
+--bootnodes=/dns4/frag3-stagenet-relay-val-2.g.moondev.network/tcp/30334/p2p/12D3KooWMAtGe8cnVrg3qGmiwNjNaeVrpWaCTj82PGWN7PBx2tth \
+--bootnodes=/dns4/frag3-stagenet-relay-val-3.g.moondev.network/tcp/30334/p2p/12D3KooWLKAf36uqBBug5W5KJhsSnn9JHFCcw8ykMkhQvW7Eus3U \
+--bootnodes=/dns4/vira-stagenet-relay-validator-0.a.moondev.network/tcp/30334/p2p/12D3KooWSVTKUkkD4KBBAQ1QjAALeZdM3R2Kc2w5eFtVxbYZEGKd \
+--bootnodes=/dns4/vira-stagenet-relay-validator-1.a.moondev.network/tcp/30334/p2p/12D3KooWFJoVyvLNpTV97SFqs91HaeoVqfFgRNYtUYJoYVbBweW4 \
+--bootnodes=/dns4/vira-stagenet-relay-validator-2.a.moondev.network/tcp/30334/p2p/12D3KooWP1FA3dq1iBmEBYdQKAe4JNuzvEcgcebxBYMLKpTNirCR \
+--bootnodes=/dns4/vira-stagenet-relay-validator-3.a.moondev.network/tcp/30334/p2p/12D3KooWDaTC6H6W1F4NkbaqK3Ema3jzc2BbhE2tyD3YEf84yNLE 
+
+[Install]
+WantedBy=multi-user.target
+```
+
+Note that the `ExecStart` command still needs some parameters to be changed before starting the service. The following section focuses on this command and its optional flags.
+
+## The `ExecStart` Command {: #execstart-command }
+
+The previous `ExecStart` example has some parameters that need to be changed to match your specific Appchain:
+
+- `EVM compatibility` - Tanssi releases two different binaries, one for EVM-compatible Appchains and another for only Substrate Appchains. Replace `APPCHAIN_TYPE` with either `frontier` for EVM Appchains or `simple` for Substrate Appchains
```

Comment:
Goes before, similar to the Docker structure. Use the exact same format so that it looks cohesive